instances/service/localiZoom.jsonld

```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/service/localiZoom",
```

Comment:
if it's a dedicated name that is capitalize you should capitalize it here as well